docs/lab/02/02.md

```diff
@@ -644,6 +644,64 @@ If the MCU provides timers, Embassy-rs will use them to suspend the software ins
 
 :::
 
+## Build and flash
+
+### Build
+
+To build a program for the Raspberry Pi Pico, run the following command (in the project root folder):
+
+`cargo build --bin <project_name>`
+
+where `<project_name>` is to be replaced with the name of your project.
+
+This will create the binary file inside `/target/thumbv6m-none-eabi/debug/`.
+
+### Flash
+
+To load your program to the Raspberry Pi Pico, connect the board through USB to your computer while holding down the `BOOTSEL` button. It should appear as an external drive on your system. Afterwards, you can run the following command (also from the root folder):
+
+`elf2uf2-rs -d .\target\thumbv6m-none-eabi\debug\<project_name>`
+
+This will upload your program to the Pico.
+
+:::note
+
+If the above command doesn't work, try these steps instead:
+
+- run this command: `elf2uf2-rs .\target\thumbv6m-none-eabi\debug\<project_name>`
+
+- drag and drop the `.uf2` file located in the `.\target\thumbv6m-none-eabi\debug\` folder to the RP external drive
+
+:::
+
+### Serial console
+
+To see messages from the Pico over the serial port, add the `-s` argument to the flash command:
+
+`elf2uf2-rs -s -d .\target\thumbv6m-none-eabi\debug\<project_name>`
+
+This will allow us to see messages sent over serial from the board.
+
+### Troubleshooting
+
+In case your code doesn't seem to get uploaded correctly or the board goes into `BOOTSEL` mode as soon as you plug it into your computer, it might be time to debug. To install required dependencies for debugging, run these commands:
+
+`rustup component add llvm-tools`
+
+`cargo install cargo-binutils`
```

Comment:
```suggestion
```shell
rustup component add llvm-tools
cargo install cargo-binutils
```
```

Reply:
Modified and added correct shell code symbol.